cli/src/commands/describe.rs

```diff
@@ -238,6 +239,15 @@ pub(crate) fn cmd_describe(
         }
     };
 
+    // Add the trailer entries to the description
+    let commit_descriptions: Vec<(_, _)> = commit_descriptions
+        .iter()
+        .map(|(commit, description)| {
+            let new_description = add_trailers(ui, &tx, commit, description)?;
+            Ok((commit, new_description))
+        })
+        .try_collect::<_, _, CommandError>()?;
```

Comment:
nit: type annotation can be moved to the `.map()` closure (or the closure can be rewritten as `add_trailers(ui, &tx, commit, description).map(|new_description| (commit, new_description))`.)